### DISCUSSION TITLE
Request help with ESP32-cam Please!!!

### DISCUSSION THREAD

**Discussion by IloveUCat** [jomjol/AI-on-the-edge-device#3412] [General]
![67783](https://github.com/user-attachments/assets/d5538560-b739-4f75-8725-d73fb60f5a1a)
### The Feature

I want an ESP32-cam that I can buy (Recommended shops) for use. AI-on-the-edge-device because in my country (Thailand) I tried to buy many of them and found that 

**I (1447) MAIN: PSRAM size: 2097152 byte (2MB / 16MBit)**
E (1457) MAIN: PSRAM size >= 4MB (32Mbit) is mandatory to run this applicationE 
E (1457) HELPER: New System Status: 0x00000001I
.
.
I MAIN: Initialization failed. Flow task start aborted. Loading reduced web interface...
I main_task: Returned from app_main()

So never had more than 2psram,I just succeeded in finding one with the correct 4psram or 8psram. I am asking for this help hoping someone can advise and help. 

Many thanks for giving me this opportunity.

**Comment by caco3** [comment]
Just use AliExpresss.
All my orders from there had the right size. But you might want to try different vendors there.
Also have a look on https://github.com/jomjol/AI-on-the-edge-device/discussions/1732

**Comment by Soufianelaw** [comment]
I want to use camera of ep32cam but after upload program I found that erreur  
rst:0x10 (RTCWDT_RTC_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
20:18:59.289 -> flash read err, 1000
20:18:59.289 -> ets_main.c 371 
20:18:59.659 -> ets Jun  8 2016 00:22:57

**Reply by caco3** [reply]
please start a new discussion and provide the full log!

**Comment by Soufianelaw** [comment]
How

**Reply by SybexX** [reply]
>How
<img width="1914" height="1002" alt="Unbenannt1" src="https://github.com/user-attachments/assets/b6d1d86a-b63d-49b4-8015-61c85448ba11" />


Maybe you have to delete the flash beforehand and then flash the firmware again.

<img width="686" height="676" alt="Unbenannt" src="https://github.com/user-attachments/assets/f4dedec6-d767-43fa-ac2c-33dca93decb8" />
